### ISSUE TITLE
Increase DotNetty package references to 0.4.7 in IoT Hub Client

### ISSUE THREAD

**Issue by CIPop** [Azure/azure-iot-sdk-csharp#250]
Increase the following to 0.4.7:

```DotNetty.Buffers, Version=0.4.6.0
DotNetty.Codecs, Version=0.4.6.0
DotNetty.Codecs.Mqtt, Version=0.4.6.0
DotNetty.Common, Version=0.4.6.0
DotNetty.Handlers, Version=0.4.6.0
DotNetty.Transport, Version=0.4.6.0
```

**Comment by CIPop** [comment]
This is causing #275.

**Comment by CIPop** [comment]
Fixed by #308.
